util/src/io/service.rs

```diff
@@ -231,8 +231,8 @@ impl<Message> Handler for IoManager<Message> where Message: Send + Clone + Sync
 	fn notify(&mut self, event_loop: &mut EventLoop<Self>, msg: Self::Message) {
 		match msg {
 			IoMessage::Shutdown => {
-				self.workers.clear();
 				event_loop.shutdown();
```

Comment:
@arkpar Could you have a look at this? Does it make any sense to shutdown `event_loop` before clearing `workers`?

`clear()` drops workers and locks until they process all work, but my guess is that some messages might still be added to the channel for them if we don't shutdown `event_loop`?